### PR TITLE
Feature/skeleton loader report

### PR DESCRIPTION
**Pull request by priyankatechneai**
Add skeleton loader in report and solved   My ticket skeleton issue

### CODE REVIEW

File: src/screens/Reports/ResourcePlanningReportComponent.jsx [hunk 1]
```diff
@@ -142,12 +132,14 @@ export default function ResourcePlanningReportComponent() {
             formData
           );
           if (res.status === 200) {
+            console.log("res", res);
```
Comment (somyaranjan-techneai) [inline src/screens/Reports/ResourcePlanningReportComponent.jsx:135]: remove console

File: src/screens/Reports/UserTaskReportComponent.jsx [hunk 2]
```diff
@@ -12,12 +12,15 @@ import ManageMenuService from "../../services/MenuManagementService/ManageMenuSe
...
 
 function UserTaskReportComponent() {
-  const [showLoaderModal, setShowLoaderModal] = useState(false);
   const dispatch = useDispatch();
+  const [showLoaderModal, setShowLoaderModal] = useState(false);
+  const [isLoading, setISLoading] = useState(false);
+
   const checkRole = useSelector((DashboardSlice) =>
-    DashboardSlice.dashboard.getRoles.filter((d) => d.menu_id == 24)
+    DashboardSlice.dashboard.getRoles.filter((d) => d.menu_id === 24)
```
Comment (somyaranjan-techneai) [inline src/screens/Reports/UserTaskReportComponent.jsx:23]: give proper name

File: src/screens/Reports/UserTaskReportComponent.jsx [hunk 3]
```diff
@@ -290,45 +264,40 @@ function UserTaskReportComponent() {
...
-                >
-                  <i className="icofont-refresh text-white"></i> Reset
-                </button>
+              <div className="row mt-4">
+                <div className="col-md-6 ">
+                  <button
+                    className="btn btn-sm btn-warning text-white"
+                    type="submit"
+                  >
+                    <i className="icofont-search-1 "></i> Search
```
Comment (somyaranjan-techneai) [inline src/screens/Reports/UserTaskReportComponent.jsx:273]: make self-closing tags

File: src/screens/Reports/UserTaskReportComponent.jsx [hunk 4]
```diff
@@ -290,45 +264,40 @@ function UserTaskReportComponent() {
...
+                  </button>
+                  <button
+                    className="btn btn-sm btn-info text-white"
+                    type="button"
+                    onClick={() => window.location.reload(false)}
+                  >
+                    <i className="icofont-refresh text-white"></i> Reset
```
Comment (somyaranjan-techneai) [inline src/screens/Reports/UserTaskReportComponent.jsx:280]: make self-closing tags

File: src/screens/Reports/VariantsReport.jsx [hunk 5]
```diff
@@ -8,20 +8,21 @@ import ReportService from "../../services/ReportService/ReportService";
...
+
 import { getRoles } from "../Dashboard/DashboardAction";
 import { useDispatch, useSelector } from "react-redux";
+import TableLoadingSkelton from "../../components/custom/loader/TableLoadingSkelton";
 
 export default function ResourcePlanningReportComponent() {
   const dispatch = useDispatch();
   const checkRole = useSelector((DashboardSlice) =>
-    DashboardSlice.dashboard.getRoles.filter((d) => d.menu_id == 38)
+    DashboardSlice.dashboard.getRoles.filter((d) => d.menu_id === 38)
```
Comment (somyaranjan-techneai) [inline src/screens/Reports/VariantsReport.jsx:19]: give proper name

File: src/screens/Reports/UserTaskReportComponent.jsx (hunk 2 above)
Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:23]: resolved

File: src/screens/Reports/UserTaskReportComponent.jsx (hunk 3 above)
Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:273]: resolved

File: src/screens/Reports/VariantsReport.jsx (hunk 5 above)
Comment (priyankatechneai) [reply src/screens/Reports/VariantsReport.jsx:20]: resolved

File: src/screens/Reports/UserTaskReportComponent.jsx (hunk 4 above)
Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:280]: resoved

Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:280]: resolve

File: src/screens/Reports/VariantsReport.jsx (hunk 5 above)
Comment (priyankatechneai) [reply src/screens/Reports/VariantsReport.jsx:20]: resolved

File: src/screens/Reports/UserTaskReportComponent.jsx (hunk 3 above)
Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:273]: resolved

File: src/screens/Reports/UserTaskReportComponent.jsx (hunk 2 above)
Comment (priyankatechneai) [reply src/screens/Reports/UserTaskReportComponent.jsx:23]: resolve

**Pull request closed without merging**